contentScript.js

```diff
@@ -0,0 +1,149 @@
+(() => {
+
+	let ytLeftControls, ytPlayer;
+	let currentVid = null;  // id of the currently playing video
+	let lastVid = null;   // id of the last handled video
+	let BmsList = [];  // list of the video bookmarks
+    
+		/* Fetch all bookmarks for current video */
+		const fetchBookmarks = () => {
+			return new Promise(resolve => {
+				// get all bookmarks for current video and create bookmark elements
+				chrome.storage.sync.get([ currentVid ], (obj) => {
+					resolve(obj[currentVid] ? JSON.parse(obj[currentVid]) : [])
+				});
+			});
+		}
+	
+		/* Clear out all previous bookmarks */
+		const delOldBms = () => {
```

Comment:
will want to strive for full word variables, so try to avoid del or bms use this instead deleteOldBookmarks(so that is reads)

Reply:
alkso please adjust naming, i will not make this comment again thank you :)